### ISSUE TITLE
Tab content partially misaligned after switching tabs

### ISSUE THREAD

**Issue by merschel** [angular/components#8588]
#### Bug, feature request, or proposal:

Bug

#### What is the expected behavior?

![bildschirmfoto 2017-11-22 um 11 37 24](https://user-images.githubusercontent.com/28409176/33123130-99f3e244-cf79-11e7-910e-a483235bfbf3.png)

#### What is the current behavior?

When switching the tabs some parts of some components are shifted.

![bildschirmfoto 2017-11-22 um 11 40 39](https://user-images.githubusercontent.com/28409176/33123259-00d23524-cf7a-11e7-87a6-9cfec7c874da.png)

If one switch the tabs and press the mouse down on the tab-button 'Tab 1' until the grey bubble animation is finished everything is fine.  

#### What are the steps to reproduce?

https://stackblitz.com/edit/angular-rarcts

but on stackblitz it seems to work fine. 

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

I use electron with:

  "dependencies": {
    "electron-log": "^2.2.11",
    "macaddress": "^0.2.8",
    "sequelize": "^4.3.2",
    "sqlite3": "^3.1.8"
  },
  "devDependencies": {
    "@angular/animations": "^5.0.2",
    "@angular/cdk": "^5.0.0-rc.1",
    "@angular/cli": "1.5.2",
    "@angular/common": "^5.0.2",
    "@angular/compiler": "^5.0.2",
    "@angular/compiler-cli": "^5.0.2",
    "@angular/core": "^5.0.2",
    "@angular/forms": "^5.0.2",
    "@angular/language-service": "^5.0.2",
    "@angular/material": "^5.0.0-rc.1",
    "@angular/platform-browser": "^5.0.2",
    "@angular/platform-browser-dynamic": "^5.0.2",
    "@angular/router": "^5.0.2",
    "@material-extended/mde": "^1.0.0-alpha.6",
    "@types/glob": "^5.0.33",
    "@types/jasmine": "2.5.45",
    "@types/node": "~7.0.48",
    "@types/serialport": "^4.0.9",
    "chokidar": "^1.7.0",
    "codelyzer": "~4.0.1",
    "core-js": "^2.4.1",
    "electron": "1.7.9",
    "electron-builder": "~19.45.5",
    "electron-reload": "^1.2.1",
    "hammerjs": "^2.0.8",
    "jasmine-core": "~2.6.2",
    "jasmine-spec-reporter": "~4.1.0",
    "jspdf": "^1.3.5",
    "karma": "~1.7.0",
    "karma-chrome-launcher": "~2.1.1",
    "karma-cli": "~1.0.1",
    "karma-coverage-istanbul-reporter": "^1.2.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "moment": "^2.19.2",
    "protractor": "~5.2.0",
    "rxjs": "^5.5.2",
    "ts-md5": "^1.2.2",
    "ts-node": "~3.3.0",
    "tslint": "~5.8.0",
    "typescript": "~2.4.2",
    "zone.js": "^0.8.4"
  }
}

**Comment by donroyco** [comment]
In which browser are you seeing this?

**Comment by merschel** [comment]
I seeing this in electron 1.7.9

**Comment by merschel** [comment]
I tried these also in firefox and chrome but it works fine. 

It seams that only in Chromium 58.0.3029.110 (the one electron is using) exist these problem.

**Comment by navjotdhanawat** [comment]
I am facing the same issue with electron.

**Comment by amcdnl** [comment]
@jelbourn - This is probably related to the issues around eagerly loading and initializing sub contents even when they aren't in the selected tab.

**Comment by amcdnl** [comment]
@merschel - Can you setup a example electron project for us to test in? It works for me in Chrome/FF/Safari.

**Comment by merschel** [comment]
Yes of course. For me it also works in Chrome/FF/Safari, but not in Chromium (58.0.3029.110). This is the one the current version of electron is using.

See  https://github.com/merschel/Material-Tab-Bug 

Or you can use Chromium and my stackblitz from above ( https://stackblitz.com/edit/angular-rarcts ).

I hope its working. Sometimes one have to switch the tabs several times.

**Comment by RicardoLimaDev** [comment]
I am seeing this too. ("electron": "1.7.5"). 
Very annoying bug.

**Comment by sreverdy** [comment]
I also got this problem. tested with Electron. version 1.7.9 and 1.7.6. 
If I quicky change tabs (tab2 => tab1 => tab2), the content is aligned.

**Comment by sreverdy** [comment]
I tested it with the last electron beta : https://github.com/electron/electron/releases/tag/v1.8.2-beta.3 
And it works as expected.

**Comment by crisbeto** [comment]
Closing due to inactivity.
